pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -50,7 +53,13 @@ type predicateMetadata struct {
 	podRequest    *schedulercache.Resource
 	podPorts      []*v1.ContainerPort
 	//key is a pod full name with the anti-affinity rules.
-	matchingAntiAffinityTerms          map[string][]matchingPodAntiAffinityTerm
+	matchingAntiAffinityTerms map[string][]matchingPodAntiAffinityTerm
+	// A map of node name to a list of Pods on the node that can potentially match
+	// the affinity rules of the "pod".
+	matchingAffinityPods map[string][]*v1.Pod
```

Comment:
probably `nodeNameToMatchingAffinityPods` conveys the intention clearly.

Reply:
Done.